yucca/task_conversion/Task299_Combine.py

```diff
@@ -27,7 +27,7 @@ def convert(path: str, subdir: str = None):
         os.path.join(target_base, "dataset.json"),
         target_imagesTr,
         imagesTs_dir=None,
-        modalities=["t1", "t2", "mprage", "flair", "gre", "dwi", "swi", "grappa"],
```

Comment:
If you want to remember which modalities are in the dataset you can also pass them as one string such as "T1, T2, MPRAGE" etc.